src/lib/TransactionSender.ts

```diff
@@ -154,7 +170,16 @@ export class TransactionSender {
   }
 
   signRawTransaction(rawTx, privateKey) {
-    const tx = new Tx(rawTx);
+    const customCommon = Common.forCustomChain(
```

Comment:
why some of these parameters are hardcoded?

Reply:
New versions of the ethereumjs-tx only accept the ethereum networks, mainnet and sepolia, so to be able to use it in RSK network need to create the customChain, those are the recommendations from the library.